ngraph/test/backend/one_hot.in.cpp

```diff
@@ -209,7 +209,15 @@ NGRAPH_TEST(${BACKEND_NAME}, one_hot_on_off_float)
     auto a = backend->create_tensor(element::i32, shape_a);
     copy_data(a,
               vector<int32_t>{
-                  0, 1, 1, 2, 1, 0, 0, 2, 1,
+                  0,
+                  1,
+                  1,
+                  2,
+                  1,
+                  0,
+                  0,
+                  2,
+                  1,
```

Comment:
Such expand might be remove by removing last (not required) coma.

Reply:
You're right, Done https://github.com/openvinotoolkit/openvino/pull/4721/commits/58b896a894f395686d7d4d878dc9dcaefaae9eaf